index.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 module.exports = class EmitAllPlugin {
     constructor(opts = {}) {
         this.ignorePattern = opts.ignorePattern || /node_modules/;
+        this.ignoreExternals = opts.externals;
```

Comment:
It would be more clear if the opts variable used the same name as the internal variable. i.e. `opts.ignoreExternals`.